src/listeners/mentionListener.js

```diff
@@ -0,0 +1,43 @@
+const { Listener } = require("discord-akairo");
+const { MessageEmbed } = require("discord.js");
+const timeUtils = require("../utilities/timeUtils");
+
+class MentionListener extends Listener {
+  constructor() {
+    super("mention", {
+      event: "message",
+      emitter: "client",
+    });
+  }
+
+  exec(message) {
+
+    if (message.author.bot) {
+        return;
+    }
+
+    if (message.mentions.users.first.id == this.client.id) {
```

Comment:
Almost perfect, it seems to react to any message not just mentions 
![image](https://user-images.githubusercontent.com/34754130/93983544-9ec88400-fd7a-11ea-9943-aa8d5bd3467a.png)

the issue is probably here. Try console logging those two values to see what they return

Reply:
Huh, that's weird, I'll fix it.